t/redirects.rb

```diff
@@ -0,0 +1,10 @@
+require 'test_helper'
+
+describe 'redirecting links' do
+  around { |test| VCR.use_cassette('liff', &test) }
+  subject { WikiData.ids_from_pages('en', ['The Meaning of Liff', 'The Deeper Meaning of Liff']) }
```

Comment:
It actually looks like multiple redirects come free with the first change anyway without changing existing code.